src/routes/living-form/containers/NeuburgForm.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-magic-numbers */
```

Comment:
I think we should add this to the few lines this is used, or change our linting rules.